### PR TITLE
Updated documentation with examples of using secrets and date_label

### PR DESCRIPTION
**Pull request by lnikell**
Hi, based on the discussions at https://github.com/josmo/drone-k8s-deployment/issues/2 
I changed a little bit existing documentation of the plugin, added example with using secrets and date_label.

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -45,4 +45,35 @@ deploy:
...
+pipeline:
+  deploy:
+    image: peloton/drone-k8s-deployment
+    token: asldkfj
+    insecure: false
+    deployment_names: [mything, mything2]
+    container_names: mything
+    namespaces: [mynamespace, anothernamspace]
+    docker_image: drone/drone:latest
+    date_label: "${DRONE_BUILD_FINISHED}"
```
Comment (josmo) [inline DOCS.md]: This should be like date_label: deployment.drone.io/date-deployed :) The date gets added to whatever annotation you use :)

**Review by josmo — CHANGES_REQUESTED** [review]
I think the only change is making the label be the key to the annotation

File: DOCS.md (hunk 1 above)
Comment (lnikell) [reply DOCS.md]: silly question, but how does it work? :) Do you mean that "deployment.drone.io/date-deployed" will automatically add the date of the deploy?
I have not seen this in drone ci documentation

Comment (josmo) [reply DOCS.md]: no, the plugin adds the current date/time to whatever label you give it (not saying it's the best implementation but it's how it currently works) might be better in the future to have a map of the key and what the value should be like DRONE_BUILD_CREATED. I just didn't do it that way yet :)

**Comment by josmo** [comment]
thanks @lnikell

**Pull request merged**